ReactWindows/ReactNative.Shared/Bridge/FileJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Threading.Tasks;
+#if WINDOWS_UWP
+using Windows.Storage;
+#else
+using System.IO;
+using System.Reflection;
+#endif
+
+namespace ReactNative.Bridge
+{
+    class FileJavaScriptBundleLoader : JavaScriptBundleLoader
+    {
+        private string _script;
+
+        public FileJavaScriptBundleLoader(string fileName)
+        {
+            SourceUrl = fileName;
+        }
+
+        public override string SourceUrl
+        {
+            get;
+        }
+
+        public override async Task InitializeAsync()
+        {
+            try
+            {
+#if WINDOWS_UWP
+                var storageFile = await StorageFile.GetFileFromApplicationUriAsync(new Uri(SourceUrl)).AsTask().ConfigureAwait(false);
+                _script = storageFile.Path;
+#else
+                var assembly = Assembly.GetAssembly(typeof(FileJavaScriptBundleLoader));
+                var assemblyName = assembly.GetName();
+                var pathToAssembly = Path.GetDirectoryName(assemblyName.CodeBase);
+                var pathToAssemblyResource = Path.Combine(pathToAssembly, SourceUrl.Replace("ms-appx:///", String.Empty));
```

Comment:
ms-appx URLs are hard-coded in several of the tests and we were trying to be less invasive since it' was easy to translate transparently and have the tests stay passing. We can absolutely make a test-helper that abstracts this, or take another approach.